.github/workflows/ci.yml

```diff
@@ -0,0 +1,105 @@
+name: Python CI
+
+on:
+  push:
+    branches:
+      - master
+    tags: ['*']
+  pull_request:
+
+concurrency:
+  group: test-${{ github.head_ref }}
+  cancel-in-progress: true
+
+env:
+  PYTHONUNBUFFERED: "1"
+  FORCE_COLOR: "1"
+
+jobs:
+  build:
+    name: Python ${{ matrix.python-version }} ${{ matrix.extra-env }}
+    if: >-
+        !contains(github.event.head_commit.message, '[skip ci]')
+        && !contains(github.event.head_commit.message, '[skip tests]')
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
This seems rather ambitious! Does it really pass on 3.11?

Reply:
According to the CI runs, yes: https://github.com/moble/galgebra/actions/runs/5823348756/job/15790024538 .

@moble Kindly remind that this PR doesn't run CI in your repo like the old PR did (which was triggered by pushing to your master, neither run as PR in the pygae repo since the CI workflow isn't in yet. You need a temporary way to trigger the CI.

Reply:
I've temporarily modified it to run on push to any branch.  I've converted this to a draft PR just to remind myself to undo that before merging.